packages/waku/src/lib/renderers/html-renderer.ts

```diff
@@ -259,12 +260,13 @@ export const renderHtml = async (
           {},
           {
             get(_target, name: string) {
-              const file = filePath.slice(config.basePath.length);
+              let file = filePath.slice(config.basePath.length);
               // TODO too long, we need to refactor this logic
               if (isDev) {
+                file = file.split('?')[0]!;
```

Comment:
Is this needed for both cases, a) startsWith('@fs/') and b) not?

Reply:
rewrote it. It's for b only.